src/scss/includes/map_theme.scss

```diff
@@ -96,4 +98,36 @@
 
 .compass-origin {
   display: none;
-}
\ No newline at end of file
+}
+
+.mapboxgl-ctrl-icon.mapboxgl-ctrl-geolocate {
+  @extend [class^="icon-"];
+  @extend .icon-pin_geoloc;
+  background-image: none !important;
+  color: $primary_text;
+  font-size: 20px;
+
+  @media(max-width: 640px) {
```

Comment:
by habit we put all media_queries together.
media queries should contains classes